packages/react/src/components/ToggleGroup/ToggleGroup.stories.tsx

```diff
@@ -65,7 +65,7 @@ export const Kontrollert: StoryFn<typeof ToggleGroup> = () => {
   const [value, setValue] = useState<string>('utkast');
   return (
     <>
-      <ToggleGroup value={value} size='md' onChange={setValue}>
+      <ToggleGroup value={value} data-size='md' onChange={setValue}>
```

Comment:
Should we maybe just remove all instance of `data-size='md'` as its the default size (unless its overriding an parent sizing)?

Reply:
I went ahead and removed it where it made sense to do so.